spec/support/fill_in_the_form_steps.rb

```diff
@@ -65,6 +65,25 @@ def and_has_given_their_date_of_birth
     end
   end
 
+  def and_has_found_their_address
+    VCR.turn_on!
+    VCR.use_cassette "/address/200_individual_building" do
+      expect(page.body).to have_content(I18n.t("coronavirus_form.questions.address_lookup.title"))
+      within find(".govuk-main-wrapper") do
+        fill_in "postcode", with: "SW1A2AA"
+        click_on I18n.t("coronavirus_form.submit_and_next")
+      end
+
+      expect(page.body).to have_content(I18n.t("coronavirus_form.questions.address_lookup.title"))
+      within find(".govuk-main-wrapper") do
+        address = "10, DOWNING STREET, LONDON, CITY OF WESTMINSTER, SW1A 2AA"
+        select(address, from: :address_string)
+        click_on I18n.t("coronavirus_form.submit_and_next")
+      end
+    end
+    VCR.turn_off!
+  end
+
```

Comment:
I think it would be useful to have tests for missing addresses and other errors here.

Reply:
Have added to other tests.